app/assets/javascripts/controllers/security_group/security_group_form_controller.js

```diff
@@ -1,54 +1,63 @@
-ManageIQ.angular.app.controller('securityGroupFormController', ['$scope', 'securityGroupFormId', 'miqService', 'API', function($scope, securityGroupFormId, miqService, API) {
-  $scope.securityGroupModel = { name: '' };
-  $scope.formId = securityGroupFormId;
-  $scope.afterGet = false;
-  $scope.modelCopy = angular.copy( $scope.securityGroupModel );
-  $scope.model = "securityGroupModel";
-
-  ManageIQ.angular.scope = $scope;
-
-  if (securityGroupFormId == 'new') {
-    $scope.securityGroupModel.name = "";
-    $scope.securityGroupModel.description = "";
-    $scope.newRecord = true;
-  } else {
-    miqService.sparkleOn();
-    API.get("/api/security_groups/" + securityGroupFormId + "?attributes=cloud_tenant").then(function(data) {
-      $scope.afterGet = true;
-      $scope.securityGroupModel.name = data.name;
-      $scope.securityGroupModel.description = data.description;
-      $scope.securityGroupModel.cloud_tenant_name = data.cloud_tenant.name;
-      $scope.modelCopy = angular.copy( $scope.securityGroupModel );
+ManageIQ.angular.app.controller('securityGroupFormController', ['securityGroupFormId', 'miqService', 'API', function(securityGroupFormId, miqService, API) {
+  var vm = this;
+
+  var init = function() {
+    vm.securityGroupModel = {
+      name: "",
+      description: "",
+      cloud_tenant_name: "",
+      firewall_rules: [],
+    };
+    vm.formId = securityGroupFormId;
+    vm.model = "securityGroupModel";
+    vm.newRecord = securityGroupFormId === "new";
+    vm.saveable = miqService.saveable;
+
+    if (vm.newRecord) {
+      vm.afterGet = false;
```

Comment:
@gildub Can you change this to `vm.afterGet = true;`, otherwise the `new` form will not load.

I assure you, this is the only change required, and then we should be done with this PR :)